templates/fine-tune-llm_v2/cookbooks/bring_your_own_data/README.ipynb

```diff
@@ -0,0 +1,89 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Bring your own data \n",
+    "**⏱️ Time to complete**: 10 minutes\n",
+    "\n",
+    "This guide focuses on how you can bring your own data to fine-tune your model on the Anyscale Platform. Make sure you've gone over the [basic fine-tuning guide](../../README.md) before going over this cookbook.\n",
+    "\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Data stored in remote storage \n",
+    "\n",
+    "## Public datasets\n",
+    "For datasets configured for public access, you simply need to add the relevant training and validation file URI in your training YAML. We support loading from data stored on S3 and GCS.\n",
+    "\n",
+    "## Private datasets\n",
+    "With private data, you have two options: \n",
+    "\n",
+    "### Option 1: Configure permissions directly in your cloud account\n",
+    "The most convenient option is to provide read permissions for your Anyscale workspace for the specific bucket. You can follow our guide to do so [here](https://docs.anyscale.com/configuration/cloud-storage-buckets#access-private-cloud-storage).\n",
```

Comment:
Let's not tell the user to even try `aws configure` I don't know if that'll mess up with the settings. just tell them to setup env variables and then start a new terminal or unset the vars.

Reply:
Hmm this is actually not too bad (i've tried this path) but I agree it might break stuff. I've removed this.